src/SFA.DAS.Commitments.Application/Queries/GetOverlappingApprenticeships/GetOverlappingApprenticeshipsQueryHandler.cs

```diff
@@ -54,7 +54,7 @@ public async Task<GetOverlappingApprenticeshipsResponse> Handle(GetOverlappingAp
 
                     if (validationFailReason != ValidationFailReason.None)
                     {
-                        _logger.Info($"ULN: {request.Uln} {request.StartDate:MMM yyyy} - {request.EndDate:MMM yyyy} Reason: {validationFailReason} " +
```

Comment:
don't think we need to remove request start and end dates

Reply:
ok. Added back